native_ref/pom.xml

```diff
@@ -27,6 +28,37 @@
           <!-- cross compiles are built separately on a Linux box... -->
 	</modules>
       </profile>
+      <profile>
```

Comment:
@fommil The purpose isn't to create a release.  It's for users who build from source.